src/app/projects/components/project-settings/project-settings.component.html

```diff
@@ -471,7 +484,7 @@
                                             (mouseenter)="setCurrentEmbeddingHandle(embeddingHandle,hoverBox,me)"
                                             (mouseleave)="setCurrentEmbeddingHandle(null)">
 
-                                            <label class="cursor-pointer label"
+                                            <label class="cursor-pointer label" [ngClass]="checkIfModelIsDownloaded(embeddingHandle.configString) ? 'text-green-700' : ''"
```

Comment:
function in html code

Reply:
(works for now so we dont change it but we need to keep an eye on this in the future)